app/controllers/EventSummaryController.scala

```diff
@@ -40,11 +40,10 @@ class EventSummaryController @Inject()(
                                  )(implicit ec: ExecutionContext) extends FrontendBaseController with I18nSupport {
 
   private val form = formProvider()
-  private val eventType = EventType.Event1
+  private val eventType = EventType.WindUp
```

Comment:
Should we remove this and add a new getData function that doesn't take an event type? Seems like it would make more sense for the event summary page